src/plots/plots.js

```diff
@@ -722,9 +722,9 @@ plots.supplyFrameDefaults = function(frameIn) {
     return frameOut;
 };
 
-plots.supplyTraceDefaults = function(traceIn, traceIndex, layout) {
+plots.supplyTraceDefaults = function(traceIn, traceOutIndex, layout, traceInIndex) {
```

Comment:
the odd call signature is to not break to toolpanel code:

![image](https://cloud.githubusercontent.com/assets/6675409/19689267/cca66e60-9a9a-11e6-9aa8-4a2ad2b5367d.png)